server/utils/roleHelpers.ts

```diff
@@ -0,0 +1,9 @@
+import { Role } from '../data/enums/role'
+
+// eslint-disable-next-line import/prefer-default-export
+export const canViewOrAddCaseNotes = (roles: string[], activeCaseLoadId: string, prisonId: string) => {
+  return (
+    (roles?.some(role => role === Role.InactiveBookings) && ['OUT', 'TRN'].includes(prisonId)) ||
+    activeCaseLoadId === prisonId
+  )
+}
```

Comment:
Just to confirm that the intended behaviour is as long as it's your active case load and not "a case load that you have"? I'm assuming it's right but just double checking :D